cmd/main.go

```diff
@@ -68,6 +68,8 @@ var (
 	tmcfCsvBucket  = flag.String("tmcf_csv_bucket", "", "The GCS bucket that contains tmcf and csv files")
 	tmcfCsvFolder  = flag.String("tmcf_csv_folder", "", "GCS folder for an import. An import must have a unique prefix within a bucket.")
 	memdbPath      = flag.String("memdb_path", "", "File path of memdb config")
+	// Remote mixer url. The API serves merged data from local and remote mixer
+	remoteMixerUrl = flag.String("remote_mixer_url", "", "Remote mixer url to fetch and merge data for API response.")
```

Comment:
For the planned future, this will only be for custom DC to talking to base DC API. I will make it a single remote for now to save the complexity downstream (concurrent fetch for multiple remotes, handle multiple APIs in config etc)